crates/signer-gcp/src/signer.rs

```diff
@@ -106,17 +106,17 @@ impl KeySpecifier {
 /// # }
 /// ```
 #[derive(Clone)]
-pub struct GcpSigner {
+pub struct GcpWallet {
```

Comment:
this is a signer, no? as it manages only a single key

Reply:
>A wallet is a credential store capable of storing multiple signers, and selecting a signer to use to sign hashes, or txns, or other data.

I was under the impression that it was more about if the device or service is capable of holding and managing multiple keys / a way to derive the next key

Reverted to use signer for these, also allows for removing the newly introduced deprecation notices

Reply:
Yeah, while GCP or the Ledger Device itself is a "wallet" according to our definition, the structs we have that manage access to those wallets function only as signers, i.e. each of these structs manage a single credential

this implies that we designed the structs wrong in the first place. the ledger struct acquiring the whole device for a single key is bad behavior. GCP instantianting a client for each key is bad behavior. We should be refactoring these from signers into wallets, but that's separate PR scope